data/hai/hai.txt

```diff
@@ -451,6 +451,16 @@ phrase "friendly hai"
 		"concord."
 		"cooperation."
 
+phrase "friendly hai"
+	word
+		"Do you have any of your human ship cooling systems to trade? Many of us consider them to be more fashionable and exotic than our Williwaws."
+		"Maelstrom isn't just a popular world for human climbers. It's a fashionable romantic getaway for our younger Hai."
```

Comment:
```suggestion
```
Before Hai Revealed, this seems questionable. ~~If only we had more dynamic hails.~~

Might be better as a Hai news item, which should be able to activate after HR, I think.